src/resources/Pipelines/MLAssociations/MLAssociationsInterfaces.ts

```diff
@@ -1,5 +1,8 @@
 import {ModelTypes} from '../../Enums';
 
+/**
+ * @deprecated - Use `PageModel<MLAssociationModel, 'rules'>` instead.
+ */
 export interface AssociationsListModel {

```

Comment:
How do you handle deprecation in the project?
i.e. when do you delete deprecate code?

Reply:
That is a good question, that I'm not sure I hold an answer for. Currently we typically remove deprecated stuff when we already intend to do a breaking change and we think about it. So its kind of random and would probably need to be more clearly defined.

Reply:
Mh then @gdostie, what is the current reason for marking the code deprecated instead of deleting it? (not that I say we should do one instead of the other, just rubber ducking on the question ;)